pages/ksphere/dispatch/1.3/references/dispatchfile/index.md

```diff
@@ -3,6 +3,6 @@ layout: layout.pug
 navigationTitle:  Dispatchfile
 title: Dispatchfile
 menuWeight: 20
-beta: true
+beta: false
```

Comment:
why do we have an empty topic here?

Reply:
I'm not sure why we had this but it's probably there for a while. I guess we could remove this.